src/components/footer/footer.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, fireEvent } from '../../test-utils';
```

Comment:
Nice abstraction of the cumbersome render with history 👍

Reply:
Thanks! I still feel like things can be simplified within my tests across the app but this is a start.